src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPartDetailAdminSearchPanel.cshtml

```diff
@@ -0,0 +1,37 @@
+@using System.Globalization
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@using OrchardCore.Lists.ViewModels
+@using OrchardCore.Localization
+
+@model OrchardCore.Lists.ViewModels.ListPartViewModel
+
+@{
+    var dropdownClassList = CultureInfo.CurrentUICulture.IsRightToLeft() 
+       ? "dropdown-menu dropdown-menu-start" 
+       : "dropdown-menu";
```

Comment:
Just have one question before you nominate me @MikeAlhayek, this time there are 4 spaces before it's by intention or VS formating?!!